README.md

```diff
@@ -7,6 +7,39 @@ hosted in a git repository.
 It makes use of the `publishConfig` section
 of a project's `package.json`.
 
+
+## `publishConfig`
+
+In your project's `package.json`,
+you can optionally include a `publishConfig` section.
+Below is an example.
+
+```
+"publishConfig": {
+  "registry": "http://some-registry.dev",
+  "license": {
+    "exclude": [
+      "lib"
+    ]
+  }
+}
+```
+
+
+### `publishConfig.registry`
+
+Specify a custom registry for use
```

Comment:
Maybe link to the npm docs here: https://docs.npmjs.com/files/package.json#publishconfig